README.md

```diff
@@ -26,8 +26,8 @@ Or install it yourself as:
 
 ## Usage
 
-With the gem, you do not need to manually add exchange rates. Calling load will
-load the rates from the cache file or download the rates from the API depending
+With the gem, you do not need to manually add exchange rates. Calling, `MoneyOXR::RatesStore#load` will
+either load the rates from the cache file or download the rates from the API and create the file depending
```

Comment:
Actually, I removed the explicit load call from the README since it's no longer required.